app/views/admin/users/show.html.erb

```diff
@@ -13,7 +13,7 @@
 
   <%= render partial: "shared/search_bar", locals: {url: admin_user_path, query: params[:search]} %>
 
-  <table id="users" class="table table-hover table-condensed">
+  <table id="users" class="table table-hover table-condensed" ng-controller="ProjectRolesCtrl" ng-init="loadProjectRoles()">
```

Comment:
You're absolutely right, I kept it from what I had previously, but there's no need to do it anymore. I'm going to change it.
Thanks for the heads up.